### PR TITLE
update lmod tests AB#21652

### PR DESCRIPTION
**Pull request by smoors**

### CODE REVIEW

File: lmod/lmod.py [hunk 1]
```diff
@@ -18,18 +18,19 @@
...
+
 
 def calc_tcgen(months):
-    "calculate the toolchain generation for a date corresponding to now + months"
+    "calculate the toolchain generation for a date corresponding to now - months"
     curtimestamp = datetime.now().timestamp()
-    newtimestamp = curtimestamp + months * 2629743  # 1 month = 2629743 seconds, as defined in SitePackage.lua
+    newtimestamp = curtimestamp - months * 2629743  # 1 month = 2629743 seconds, as defined in SitePackage.lua
     newdate = date.fromtimestamp(newtimestamp)
-    suffix = 'a' if newdate.month < 7 else 'b'
```
Comment (wpoely86) [inline lmod/lmod.py:27]: why this change?

Comment (smoors) [reply lmod/lmod.py:27]: from 2022a on we're only installing the 'a' toolchains in hydra

**Review by wpoely86 — APPROVED** [review]
